packages/typescript-compat/v4.9.x/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@connectrpc/connect-conformance": "*",
     "@connectrpc/example": "*",
-    "@types/node": "22.9.0",
+    "@types/node": "22.10.1",
```

Comment:
The `@types/node` package has tags for the TypeScript versions it's compatible with. Looks like we can only update to 22.9.3 here. See https://www.npmjs.com/package/@types/node?activeTab=versions